vercel.json

```
@@ -2,8 +2,12 @@
   "buildCommand": "pnpm turbo build",
   "crons": [
     {
-      "path": "/api/cron/fluid-proposal",
+      "path": "/api/cron/fluid-proposal/sync",
       "schedule": "0 * * * *"
+    },
+    {
+      "path": "/api/cron/fluid-proposal/syncSupertoken",
+      "schedule": "* * * * *"
```

It's will run each minute maybe we want it to run each 1 hour, the suggestion below its how it works.

```suggestion
      "schedule": "0 * * * *"
```

In fact maybe we want even more time between, maybe days. @0xGabi can tell us what could be ideal?

---

Sure, I should've been asked before, let's see what @0xGabi will suggest

---

Hey, I think we can update the sync method to be 6 hours and the syncSupertoken depends on which parameters we end using nevertheless something like 2-5 days sounds a good first estimation